src/framework/db/test_support.clj

```diff
@@ -17,34 +18,54 @@
                       :framework.db.storage/postgresql
                       (assoc
                         :port pg-port
-                        :embedded pg
+                        :embedded (reify AutoCloseable
```

Comment:
@g-krisztian could you tell me if (.stop pg) works for embedded-postgres?

Reply:
It's easier than you think, it's:
`public class EmbeddedPostgres implements Closeable`